docs/APIContract.md

```diff
@@ -0,0 +1,67 @@
+# Versions
+
+So far there is only one version
+
+The base URL for an API should be: _baseURL_/_apiVersion_/ meaning even something such as sometestbuildsforpoe.gg/something/a/b/c/ would be valid as a base URL, as long  as it has afterwards the per version required endpoints.
+
+## Metadata Endpoint
+
+In order to provide a consistent API a metadata endpoint should be provided under _baseURL_/.well-known/pathofbuilding this endpoint should provide information about the API and the version of the API.
+Furthermore this may allow having custom endpoints for the API.
+
+| Feature          | Field         | Type | Description                                                                       |
+| ---------------- | ------------- | ---- | --------------------------------------------------------------------------------- |
+| League Filtering | league_filter | bool | This can  be used to indicate whether the API supports filtering based on leagues |
+| Gem Filtering    | gem_filter    | bool | This can  be used to indicate whether the API supports filtering based on gems    |
+
+## Version 1
+
+### Response and Request types
+
+#### BuildInfo
+
+| Field        | Type    | Description                         |
+| ------------ | ------- | ----------------------------------- |
+| pobdata      | string  | The Path of Building data           |
```

Comment:
What does this contain, the build code?

Reply:
Yes or rather the standard pob code used for sharing builds, so I assume the description should make this more clear?